src/index.ts

```diff
@@ -6,729 +6,724 @@ import { STORYBLOK_AGENT, STORYBLOK_JS_CLIENT_AGENT } from './constants'
 
 import Method from './constants'
 import {
-	ISbStoriesParams,
```

Comment:
Note that there is some noise in the diffs here from formatting changes (from formatting the file with Prettier). Once https://github.com/storyblok/storyblok-js-client/pull/831 is merged I can rebase to make the diffs quieter.

Reply:
In the meantime - you should "Hide whitespace" in the Github view settings above.

Reply:
Hey, @ryami333 I actually added a suggestion in your previous PR so we can get rid of that noisy whitespace indentation. Once its merged its gonna make this more easy to review

Reply:
Yes, I will rebase this PR as soon as that one is merged.

Reply:
@alvarosabu rebased onto `main`, ready for review.

Reply:
Hi @ryami333 thanks for taking care of it. One thing about rebasing. We had problems in the past with our automatic semantic release not grabbing the correct versions due to the use of rebase. In general, use merge commits instead. This ensures that the history remains linear and clear, allowing the semantic release tool to accurately determine the correct versions based on the commit messages.